src/internal/connector/sharepoint/restore.go

```diff
@@ -76,6 +76,8 @@ func RestoreCollections(
 				deets,
 				errUpdater,
 			)
+		case path.PagesCategory:
+			logger.Ctx(ctx).Infof("backup of pages not supported", dc.FullPath().Category())
```

Comment:
nit: I'd mark this as a warning or error level log message so it's easier to find

Reply:
I vote for error level, and returning an error along with it.  Warning level logs should be avoided.

Reply:
Resolved within [c25a1ae](https://github.com/alcionai/corso/pull/2114/commits/c25a1ae3c6f816e70058362e0d022369a46af28e)